util/util.go

```diff
@@ -57,3 +58,19 @@ func WriteTeriminateMessage(message string) {
 		panic(err)
 	}
 }
+
+// Merge the two parameters Slice and overwriting duplicate keys,
+func MergeParameters(params ...[]wfv1.Parameter) []wfv1.Parameter {
+	resultParams := make([]wfv1.Parameter, 0)
```

Comment:
This is clearer, imo
```suggestion
	var resultParams []wfv1.Parameter
```